code/modules/species/species.dm

```diff
@@ -176,13 +169,17 @@
 	/// The likelihood of a speech sound playing.
 	var/list/speech_chance = list()
 
-	//// Age
+	//? Abilities
```

Comment:
doesnt this replace inherent spells, and maybe inherent verbs, if yes please move it to that block (~line 320)